src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/ConfigurationTest.php

```diff
@@ -259,7 +260,7 @@ class_exists(SemaphoreStore::class) && SemaphoreStore::isSupported() ? 'semaphor
                         'entity_manager_name' => null,
                     ),
                     'validation' => array(
-                        'enabled' => false,
+                        'enabled' => !class_exists(FullStack::class) && class_exists(Validation::class),
```

Comment:
Should only be `!class_exists(FullStack::class)` as for other configurations. Which makes sense as the Validator component is always available in tests, being part of the `require-dev` section :)

Reply:
Hehe. I was pushing this change. Done 🙃